.github/workflows/sonar.yaml

```diff
@@ -0,0 +1,41 @@
+#
+#    Copyright 2010-2020 the original author or authors.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+name: SonarCloud
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    if: github.repository_owner == 'mybatis'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Disabling shallow clone is recommended for improving relevancy of reporting
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Analyze with SonarCloud
+        run: ./mvnw verify sonar:sonar -B -Dsonar.projectKey=mybatis_migrations-maven-plugin -Dsonar.organization=mybatis -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN -Dlicense.skip=true
```

Comment:
Hey Jeremy - you need to add the "jacoco:report" goal here to get code coverage into Sonar.

Reply:
Hi, I'm adding it through the parent to fix so all the builds auto pick it up.  I need to do some spot check with checkstyle before releasing that to make sure nothing broke on the version updates but only about 3 months since last parent.  I should have that sometime later today. For the most part, only 2 projects were actually submitting to sonar before going this direction.  I shoud have that squared away entirely by end of the week.

Reply:
see https://github.com/mybatis/parent/commit/58fd0cbdfc962cea814b5716909eebb683baa818

While this adds an extra execution, I've been doing so on a larger scale for some time and not noticed any real penalty for that execution.  Given jacoco.exec is already created, the report isn't much more beyond that.